### DISCUSSION TITLE
Fund Me forge install error code 128

### DISCUSSION THREAD

**Discussion by jpmatenga** [Cyfrin/foundry-full-course-cu#2770] [General]
Hi I'm trying to forge install the smartcontractkit/chainlink-brownie-contracts but I am getting the following: 
```
Error: git submodule exited with code 128. 
```
I have googled and tried to use AI for solutions but i cant seem to resolve the issue. What am I doing wrong? i'm really enjoying the course and am not ready to give up just yet lol

**Comment by EngrPips** [comment]
Hello @jpmatenga, Are you sure your internet connection is stable?

**Reply by jpmatenga** [reply]
yes my connection is stable and i have tried 2 different locations

**Reply by EngrPips** [reply]
What did you find when you browsed the error?

**Reply by jpmatenga** [reply]
Based on the error message and the search results, it seems you're encountering an issue with installing the smartcontractkit/chainlink-brownie-contracts using Forge. Here are some steps to troubleshoot and potentially resolve the issue:

Check the .gitmodules file: Run the command git restore .gitmodules to ensure the .gitmodules file exists in your working directory [1](https://github.com/Cyfrin/foundry-full-course-cu/discussions/964).
Verify submodule paths: Make sure the submodule name and path specified in the .gitmodules file match exactly with the actual name and path of the submodule [1](https://github.com/Cyfrin/foundry-full-course-cu/discussions/964). For example:
[submodule "lib/forge-std"]
  path = lib/forge-std
  url = https://github.com/foundry-rs/forge-std
  branch = v1.5.3
Try removing the version specifier: Instead of specifying a version, try installing without a version number [1](https://github.com/Cyfrin/foundry-full-course-cu/discussions/964):
forge install smartcontractkit/chainlink-brownie-contracts --no-commit
Create a new project: As a workaround, you can create a new project using forge init and then copy-paste your existing .sol files into the new project [1](https://github.com/Cyfrin/foundry-full-course-cu/discussions/964).
Check network connectivity: Ensure you have a stable internet connection, as the error message suggests a "RPC failed" issue [1](https://github.com/Cyfrin/foundry-full-course-cu/discussions/964).
Update Forge: Make sure you're using the latest version of Forge. You can update it using:
forge update
Clear cache: Try clearing Forge's cache:
forge clean

**Reply by EngrPips** [reply]
> Hello @jpmatenga, Are you sure your internet connection is stable?

I said it.

**Reply by jpmatenga** [reply]
yes you did lol. i use a starlink connection, it failed with that, then i tried fibre connection. Lastly tried a mobile connection and that worked lol

**Reply by EngrPips** [reply]
Interesting. That is some troubleshooting right there.

**Reply by AupikZupik** [reply]
> yes you did lol. i use a starlink connection, it failed with that, then i tried fibre connection. Lastly tried a mobile connection and that worked lol


ahahahahah
Same Here, What kind of problem is this? 
Thanks for the Simple solution.

**Reply by EngrPips** [reply]
It is what it is, hahaha.

**Comment by Totiruzi** [comment]
Hello @jpmatenga  did you add the --no-commit flag to your command?

**Reply by jpmatenga** [reply]
yes. this is what i typed forge install smartcontractkit/chainlink-brownie-contracts@1.3.0 --no-commit

**Comment by jpmatenga** [comment]
Thank you everyone it finally worked. it seems the issue was my internet connection. I used a mobile connection and it worked

**Comment by Totiruzi** [comment]
Great you figured it out )
keep thriving mate )
